examples/06_datetime_encoder.py

```diff
@@ -0,0 +1,103 @@
+"""
+Handling datetime features with the DatetimeEncoder
+==========================================
+
+We illustrate here how to handle datetime features with the DatetimeEncoder. The DatetimeEncoder
+breaks down each datetime features into several numerical features, by extracting relevant information from the
+datetime features, such as the month, the day of the week, the hour of the day, etc. Used in
+the SuperVectorizer, which automatically detects the datetime features, the DatetimeEncoder allows
+to handle datetime features easily.
```

Comment:
Would it be possible to do a small prediction model at the end, add a section, to showcase this in a prediction pipeline?

For didactic purposes it would be important to have not only the cross-validation (using https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.TimeSeriesSplit.html ) but also the plot of the extrapolated time series (compared to the real one)